specifyweb/specify/management/commands/base_specify_migration.py

```
@@ -25,28 +25,19 @@ def add_arguments(self, parser):
     def handle(self, *args, **options):
         use_override = bool(options.get('use_override', False))
         alias = options["database"]
-        conn = connections[alias]
         logger.info(f"Running base_specify_migration using database alias '{alias}'")
 
+        # Validate the alias exists and is usable; fallback to 'master' if not
         try:
-            transaction.atomic(using=alias)
-        except:
+            connections[alias].ensure_connection()
+        except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)